layout/js/gages.js

```diff
@@ -107,4 +107,11 @@ var hovertext;
       mapSVG.hideTooltip();
     }
   }
+  
+//  vizlab.showline = function(year) {
+//     var lineId = '#l' + year;
+//     document.getElementById(lineId).setAttribute(class, 'selected-year'//);
```

Comment:
One advantage of using jquery and addClass/removeClass is that the function only adds/removes the class string specified. Other classes are left in place. So on the other svg, all of the gage rectangles have class of .gage-count-bar. I add the class "selected-year" to give the red styling. And then remove it when the bar is no longer hovered over. To do the same with straight js, you would need to retrieve the class and add the new one separating the two classes with a space.